data/countries/PK.yaml

```diff
@@ -0,0 +1,46 @@
+holidays:
+  PK:
+    names:
+      en: Pakistan
+    langs: []
+    zones:
+      - Asia/Karachi
+    dayoff: sunday
+    days:
+      02-05:
+        type: observance
+        name:
+          en: Kashmir Solidarity Day
+      03-23:
+        type: observance
+        name:
+          en: Pakistan Day
+      02-01:
+        type: observance
+        name:
+          en: Labour Day
+      08-14:
+        type: observance
+        name:
+          en: Independence Day of Pakistan
+      12-25:
+        type: observance
+        name:
+          en: Quaid-e-Azam's Birthday
+    # states:
```

Comment:
Would it be possible for you to add also muslim holidays? You can find them here https://github.com/commenthol/date-holidays/blob/master/docs/specification.md#hijra-dates